frontend/src/v5/store/users/users.sagas.ts

```diff
@@ -22,15 +22,16 @@ import { DialogsActions } from '@/v5/store/dialogs/dialogs.redux';
 import { formatMessage } from '@/v5/services/intl';
 import { FetchUsersAction, UsersActions, UsersTypes } from './users.redux';
 import { selectUsersByTeamspace } from './users.selectors';
+import { getMemberImgSrc } from './users.helpers';
 
 export function* fetchUsers({ teamspace }: FetchUsersAction) {
 	try {
 		const users = yield select(selectUsersByTeamspace, teamspace);
 
 		if (isEmpty(users)) {
 			const { data: { members } } = yield API.Users.fetchTeamspaceMembers(teamspace);
-
-			yield put(UsersActions.fetchUsersSuccess(teamspace, members));
+			const membersWithImg = members.map((member) => ({ ...member, avatarUrl: getMemberImgSrc(teamspace, member.user), hasAvatar: true }));
```

Comment:
I think in general this is not needed. It should be done only in the avatar component files.
I created an issue to tackle this in the future
https://github.com/3drepo/3drepo.io/issues/4232